api/v1alpha1/obtenant_types.go

```diff
@@ -113,8 +113,9 @@ type OBTenantStatus struct {
 	OperationContext *OperationContext    `json:"operationContext,omitempty"`
 	TenantRecordInfo TenantRecordInfo     `json:"tenantRecordInfo,omitempty"`
 
-	TenantRole constants.TenantRole `json:"tenantRole,omitempty"`
-	Source     *TenantSourceStatus  `json:"source,omitempty"`
+	TenantRole  constants.TenantRole `json:"tenantRole,omitempty"`
+	Source      *TenantSourceStatus  `json:"source,omitempty"`
+	Credentials TenantCredentials    `json:"credentials,omitempty"`
```

Comment:
maybe it's better to use pointer type for Credentials since it's optional

Reply:
Zero value of type `TenantCredentials` is useful in a way, so it's reasonable to use plain type